assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
I would opt for having an oocss class of "unstyled-list" over changing the default behavior of a list.  the same would go with inline lists as well.